client/mds3/sampletable.js

```diff
@@ -75,8 +76,25 @@ export async function displaySampleTable(samples, args) {
 	const params = { rows, columns, div: args.div, resize: rows.length > 10 }
 	//if (args.maxWidth) params.maxWidth = args.maxWidth
 	//if (args.maxHeight) params.maxHeight = args.maxHeight
+
+	params.columnButtons = []
```

Comment:
columnButtons should be conditional; only set if there are buttons to add for a dataset
for ash dataset it generates a blank column with no buttons as the dataset does not have single sample data

Reply:
Fixed now